packages/react-composites/tests/browser/call/Localization.test.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { test } from './fixture';
+import { dataUiId, loadCallCompositePage, waitForCallCompositeToLoad } from '../utils';
+import { expect } from '@playwright/test';
+
+test.describe('Localization tests', async () => {
```

Comment:
Open question - these tests don't perform any visual snapshot; is this adding more value than the jest tests we have for localization already?
Instead of pulling the text from the html should these be a visual snapshot instead?

Reply:
I chose to just look at text because I thought that is the main concern of localization. Also did not want to introduce more flakiness. Although 

> > Open question I'm not sure the answer on - these tests don't perform any visual snapshot; is this adding more value than the jest tests we have for localization already?
> > Instead of pulling the text from the html should these be a visual snapshot instead?
> 
> I'd prefer these to be snapshot tests, for the reasons above. We can and should keep more detailed tests as unittests.

Ok, sounds good. Changed to snapshot tests instead.